package.json

```
@@ -29,6 +29,7 @@
   },
   "devDependencies": {
     "@massalabs/prettier-config-as": "^0.0.2",
+    "@massalabs/eslint-config": "^0.0.8",
```

```suggestion
    "@massalabs/eslint-config": "^0.0.9",
```

then run npm install to update lock file